src/mongo_odm/query_builder.hpp

```diff
@@ -68,7 +68,7 @@ class ComparisonExpr {
     operator bsoncxx::document::view_or_value() const {
```

Comment:
I replied to your comment on this from Friday; as far as I understand it, this would require multiple implicit conversions from `ComparisonExpr -> value -> view_or_value` to pass into the collection methods, which the compiler can't do. At any rate, I get syntax errors "no viable conversion from CompairsonExpr to view_or_value..." when I make this change.